src/seeders/20181217155521-testUsers.js

```diff
@@ -0,0 +1,34 @@
+
+module.exports = {
+  up: queryInterface => queryInterface.bulkInsert('Users', [{
+    id: 'dccd8ee7-bc98-4a8e-a832-ca116c5fff0a',
+    firstName: 'John',
+    lastName: 'Doe',
+    email: 'yomizy@wizzy.com',
+    roleId: 1,
+    password: 'myPassword',
+    username: 'icecream',
+    createdAt: new Date(Date.now()),
+    updatedAt: new Date(Date.now())
+  }], {}),
+  /*
```

Comment:
Change filename to match naming convention; Remove comments

Reply:
The filename has been changed and all comments removed.